include/CLI/Option.hpp

```diff
@@ -627,6 +627,8 @@ class Option : public OptionBase<Option> {
     std::string get_name(bool positional = false, //<[input] Show the positional name
                          bool all_options = false //<[input] Show every option
                          ) const {
+        if (detail::to_lower(get_group()).empty())
```

Comment:
Don't think we need `detail::to_lower()` call.

Reply:
wouldn't appear so if we are just checking empty()